README.md

```diff
@@ -31,6 +47,10 @@ Run a dev build:
 -   `yarn suite:dev:android` (react-native Android)
 -   `yarn suite:dev:ios` (react-native iOS)
 
+## trezor-connect development
+
+Trezor Connect is a platform for easy integration of Trezor into 3rd party services. At the moment, trezor-connect has its [own repository](https://github.com/trezor/connect) but we expect it to be migrated into the trezor-suite monorepo in a very short time.
+
```

Comment:
added trezor-connect development sections. My intention is to document here how to contribute to trezor-connect in the timeframe between migration start to its end.